src/crossReference/crossReference.js

```diff
@@ -81,13 +81,20 @@ const create = context => {
       return jsModule;
     }
 
+    // Checks to see if the import path is a namespace within the application or an external node module
+    const isImportPathAnAppNamespace = config.modules.find(moduleName =>
+      moduleName.startsWith(importPath.split("/")[0])
+    );
```

Comment:
This is the fix itself. I compare the import path with the computed valid aliases. Rather below it's the other way around.

So basically we compare if `@remix` startsWith `@remix-run` rather than if `@remix-run` starts with `@remix`. I had to keep the other comparisons as is due to some aliases containing nested paths and needing to maintain those. 

Ultimately I think this cross-reference rule needs some new love and a complete refactor but this should patch us up until that time.